apps/studio/components/interfaces/Auth/ThirdPartyAuthForm/AddIntegrationDropdown.tsx

```diff
@@ -0,0 +1,56 @@
+import { ChevronDown } from 'lucide-react'
+import Image from 'next/legacy/image'
+
+import {
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from 'ui'
+import {
+  INTEGRATION_TYPES,
+  getIntegrationTypeIcon,
+  getIntegrationTypeLabel,
+} from './ThirdPartyAuthForm.utils'
+
+interface AddIntegrationDropdownProps {
+  buttonText?: string
+  onSelectIntegrationType: (type: INTEGRATION_TYPES) => void
+}
+
+const Providers: INTEGRATION_TYPES[] = ['firebase', 'auth0', 'awsCognito']
+
+export const AddIntegrationDropdown = ({
+  onSelectIntegrationType,
+}: AddIntegrationDropdownProps) => {
+  return (
+    <DropdownMenu modal={false}>
+      <DropdownMenuTrigger asChild>
+        <Button type="primary" iconRight={<ChevronDown className="w-4 h-4" strokeWidth={1} />}>
```

Comment:
guessing a size 14 should work just fine?
```suggestion
        <Button type="primary" iconRight={<ChevronDown size={14} strokeWidth={1} />}>
```

Reply:
Fixed.